onnxruntime/core/providers/cuda/tensor/tile_impl.cu

```diff
@@ -7,140 +7,178 @@
 namespace onnxruntime {
 namespace cuda {
 
-constexpr int MAX_DIMS = 16;
+#ifdef USE_ROCM
+constexpr int num_elements_per_thread = 2;
+constexpr int num_threads_per_block = 512;
```

Comment:
is this following existing ROCM coding pattern? 

@PeixuanZuo

Reply:
I assume these number difference are based on some experiments as some other kernels are doing this?